### PR TITLE
Issue Template format

### PR DESCRIPTION
**Pull request by elishaROBINSON**
fix issue template format into a .github/ISSUE_TEMPLATE folder.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/standard_issue_template.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+---
+name: Standard Issue Template
+about: A Template describing the standard issue format  
+title: Standard Issue Template
+labels: bug
```
Comment (eddiejaoude) [inline .github/ISSUE_TEMPLATE/standard_issue_template.md]: Not sure if `bug` is right here?

**Comment by eddiejaoude** [comment]
Thanks @elishaROBINSON . Done as part of a live stream https://youtu.be/cryscnTdqMw

**Comment by elishaROBINSON** [comment]
git-hub supports issue templates for different types of bugs based on labels. It is i believe standard practice now to put a issue template specific to the label.  of-course the current issue template works just fine as you pointed out. you already know this but here is a link to github issue template doc https://help.github.com/en/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository . Thanks :+1:

**Comment by atodorov** [comment]
@elishaROBINSON thanks for your efforts.

I see the point in having multiple issue templates when we have different types of issue reports. ATM this project doesn't differentiate between the types of issues reported, all of them fall into the same category. Hence this change is not needed.

**Pull request closed without merging**